mods/player_api/api.lua

```diff
@@ -135,3 +145,43 @@ minetest.register_globalstep(function(dtime)
 		end
 	end
 end)
+
+-- player get node vars
+local update_time = 1.0
+local timer = 0
+
+minetest.register_globalstep(function(dtime)
+
+	timer = timer + dtime
+	if timer < update_time then
+		return
+	end
+	timer = 0
+
+	local name, pos, ndef
+
+	for _,player in pairs(minetest.get_connected_players()) do
```

Comment:
Space here.  Also why should there be so many blank lines?

Reply:
I do like my code to be readable and not scrunched together, but will look into removing white space more.